Automattic/jetpack
# Protect: Sync user data

#### Description
Currently, Protect does not include the User sync module in the sync config generated on activation. In the paid version we process a number of requests that require this module in order to pass capabilities checks. Without access, the ignore threats request and access the Cloud UI both produce `401 Unauthorized` errors. Additionally, the absence of the User module causes a looping issue in the Cloud/Calypso UI when adding credentials for what appears to be the same reason.

#### Changes proposed in this Pull Request:

* Includes the User sync module in the initial sync of data to the cache site upon plugin activation

#### Other information:

#### Jetpack product discussion


- 1201069996155224-as-1203207997801686
- peb6dq-7C-p2#comment-99

#### Does this pull request change what data or activity we track or use?


- No

#### Testing instructions:





* Start up a fresh Jurassic Ninja instance with Jetpack Beta running this Protect branch
* Add a known security threat
* Approve a user connection and add a Scan level plan
* Queue a scan to detect the threat
* Verify that Ignore threat processes the action successfully
* Proceed to the Cloud link provided in the Protect admin footer
* Verify that you are able to access the Cloud and that your site connection is valid
* Proceed to Cloud > Settings and enter your server credentials, verify that the entry succeeds and that a single scan is processed
